HousingResidentInformationAPI.Tests/V1/UseCase/ValidatePostcodeTests.cs

```diff
@@ -0,0 +1,42 @@
+using HousingResidentInformationAPI.V1.UseCase;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace HousingResidentInformationAPI.Tests.V1.UseCase
+{
+    public class ValidatePostcodeTests
+    {
+        [TestCase("E8 1JJ")]
+        [TestCase("E8T 1JJ")]
+        [TestCase("EY58 1JJ")]
```

Comment:
What does this do?

Reply:
This is a test for the postcode format validation regular expression. Incorrectly formatted postcodes will return a 400